lib/sidekiq_stats_instrumentation/client_middleware.rb

```diff
@@ -5,7 +5,7 @@
 module SidekiqStatsInstrumentation
   class ClientMiddleware
     def call(worker_class, _job, _queue, _redis_pool)
-      klass = Object.const_get(worker_class)
+      klass = worker_class
```

Comment:
this change caused 

http://sentry.vfs.va.gov/organizations/vsp/issues/40527/?project=3&query=transaction%3ASidekiq&statsPeriod=14d